pkg/kubelet/cm/helpers_linux.go

```diff
@@ -25,7 +25,7 @@ import (
 // GetCgroupSubsystems returns information about the mounted cgroup subsystems
 func GetCgroupSubsystems() (*CgroupSubsystems, error) {
 	// get all cgroup mounts.
-	allCgroups, err := libcontainercgroups.GetCgroupMounts()
```

Comment:
no.  basically, the reason we needed to pass true in cadvisor is the same reason we need to pass true in kubelet.  we need to ensure we have the right mount points (from the host) and not local to the container.  this code is not really activated until kube 1.5 with the pod level cgroup work i am doing.